Libraries/LibWeb/HTML/Scripting/Environments.h

```diff
@@ -138,6 +139,21 @@ struct WEB_API EnvironmentSettingsObject : public Environment {
 
     virtual void discard_environment() override;
 
+    // FIXME: This method below is from HighResolutionTime spec in section 3. Section for Specification Authors.
+    // There are other methods that are not yet implemented in ladybird.
+
+    // https://w3c.github.io/hr-time/#dfn-eso-current-wall-time
+    HighResolutionTime::DOMHighResTimeStamp current_wall_time() const
+    {
+        // An environment settings object settingsObject's current wall time is the result of the following steps:
+
+        // 1. Let unsafeWallTime be the wall clock's unsafe current time.
+        HighResolutionTime::DOMHighResTimeStamp unsafeWallTime = HighResolutionTime::wall_clock_unsafe_current_time();
```

Comment:
Please use `snake_case` for variable names, even ones from the spec. eg, `unsafe_wall_time`.

Also nit: We tend to use `auto` for things where the type is obvious.